### ISSUE TITLE
Low classification accuracy of Resnets?

### ISSUE THREAD

**Issue by wielandbrendel** [pytorch/vision#201]
I am trying to use the pretrained resnet models but the classification accuracy is < 10%. The same preprocessing works fine with the pretrained alexnet (Top-1 > 50%). Here is my script (the target label is part of the image name in my configuration, all images are taken from the Imagenet validation set):

```
import torchvision.models as models
from foolbox.models import PyTorchModel
import tqdm

net = models.resnet152(pretrained=True).cuda()

def preprocess_fn(x):
    x = x / 255
    x -= np.array([0.485, 0.456, 0.406])[None,:,None,None]
    x /= np.array([0.229, 0.224, 0.225])[None,:,None,None]
    return x.copy()

import glob
images = glob.glob("images/imagenet/*.png")

top1 = 0

for image_path in tqdm.tqdm(images):
    image = Image.open(image_path)
    image_size = 300
    image = image.resize((image_size, image_size), Image.ANTIALIAS)
    image = np.array(image).astype(np.float32)
    image = np.transpose(image, [2,0,1])
    label = int(image_path.split('.')[0].split('_')[1])
    
    image = torch.from_numpy(preprocess_fn(image[None])).cuda()
    image = Variable(image, volatile=True)
    q = net(image).data.cpu().numpy()[0]
    
    top1 += np.argmax(q) == label
    
print(top1 / len(images))
```

**Comment by wielandbrendel** [comment]
Closing in favour of #222 .
